pages/book/operators.mdx

```diff
@@ -29,7 +29,7 @@ Even though this example may be simple, neglection of precedence rules can often
 
 ## Parentheses, `()`
 
-Parentheses (also can be called round brackets, `(){:tact}`) are more of a punctuation symbols than actual operators, but their [precedence](#precedence) is higher than precedence of any other operator. Use parentheses to override order of operations:
+Parentheses (also can be called round brackets, `(){:tact}`) are more of a punctuation symbols than actual operators, but their [precedence](#precedence) is higher than precedence of any other operator or [expression](/book/expressions). Use parentheses to override order of operations:
```

Comment:
Same issue as in the comment about operators vs expressions

Reply:
Yup, will remove those too, like there: https://github.com/tact-lang/tact-docs/pull/136#discussion_r1534565699